frontend/src/components/Register.tsx

```diff
@@ -18,8 +18,11 @@ const Register = () => {
   async function sendRequest() {
     try {
       if (authInputs.name && authInputs.email && authInputs.password) {
-        await axios.post(`${BACKEND_URL}/api/v1/user/signup`, authInputs);
-        navigate("/signin");
+        const response = await axios.post(`${BACKEND_URL}/api/v1/user/signup`, authInputs);
+        const token = response.data.jwt;
```

Comment:
Please check the backend code for `/signup` API endpoint. 
I believe it does not return jwt / user data in the response.

Reply:
Hey i make some changes in code 
![image](https://github.com/aadeshkulkarni/medium-app/assets/119005879/ae826fdb-34ff-49b4-8117-f0d44f09a5b6)